docs/serving/autoscaling/concurrency.md

```diff
@@ -97,6 +100,19 @@ spec:
     spec:
       containerConcurrency: 50
 ```
+{{< /tab >}}
+{{% tab name="Global (Defaults ConfigMap)" %}}
```

Comment:
Operator example too? :pray:

Reply:
heh, never actually used the operator ☺️, but took an educated guess